grpc/src/main/java/com/linecorp/armeria/internal/server/grpc/AbstractServerCall.java

```diff
@@ -257,14 +257,14 @@ private void doClose(Status status, Metadata metadata, @Nullable Throwable excep
         doClose(status, metadata, completed);
     }
 
-    abstract void doClose(Status status, Metadata metadata, boolean completed);
+    public abstract void doClose(Status status, Metadata metadata, boolean completed);
 
-    final void closeListener(Status newStatus, Metadata metadata, boolean completed,
+    public final void closeListener(Status newStatus, Metadata metadata, boolean completed,
                              boolean setResponseContent) {
```

Comment:
```suggestion
    protected final void closeListener(Status newStatus, Metadata metadata, boolean completed,
                                       boolean setResponseContent) {
```

Reply:
Done in [fa4d628](https://github.com/line/armeria/commit/fa4d628b0a113437da1d9ebb625332dad3b299be)!